doc/make_cite_html.py

```diff
@@ -0,0 +1,115 @@
+# A python3 script to generate the database.js file from manual/manual.bib for citing.html online
+
+# What this does:
+# - read bibtex entries from "manual/manual.bib" that are specified below
+# - use the DOI and the online API to request a clean text form for citation for each entry
+# - write database.js (in the current directory) that contains a javascript object with all the information above
+#
+# Usage: python3 make_cite_html.py
+
+import requests
+import re
+from cgi import escape
+
+bibfile = "manual/manual.bib"
+
+bibitems = {}
+biburls = {}
+with open(bibfile) as f:
+    content = f.readlines()
+
+entry = []
+name = ""
+openbraces = 0
+for line in content:
+    line = line.strip('\n').strip(' ')
+    if len(line)==0:
+        continue
+
+    if openbraces == 0:
+        match = re.match(r"\@(\w+)\s*\{([^ ,]+).*", line)
+        if not match:
+            print("error could not match: '{}'".format(line))
+        name = match.group(2)
+        #print "found {} with key {}".format(match.group(1), name)
+        openbraces = 0
+        entry = []
+
+    openbraces = openbraces + line.count('{') - line.count('}')
+    
+    if openbraces < 0:
+        print ("ERROR!")
+        openbraces = 0
+    entry.append(line)
+
+    # find URL
+    match = re.match(r"\s*[uU][rR][lL]\s*=\s*\{([^,]+)", line)
+    if match:
+        url = match.group(1)
+        url = url.strip(',').strip('}')
+        biburls[name] = url
+
+    bibitems[name] = entry
+
+
+want_groups = {"KHB12": "main",
+               "heister_aspect_methods2": "main",
+               "aspect-doi-v1.5.0" : "1.5.0",
+               "aspect-doi-v2.0.0" : "2.0.0",
+               "aspect-doi-v2.0.1" : "2.0.1",
```

Comment:
we will have to update these lists for every release. Can you add an entry in the release tasklist so we remember?